### ISSUE TITLE
The 2nd argument of FileWriter.add_graph() in docs should be input variable instead of graph output.

### ISSUE THREAD

**Issue by hsluoyz** [lanpa/tensorboardX#228]
My env:

```
C:\Users\xxx>python3 -V
Python 3.6.6

C:\Users\xxx>pip3 freeze
absl-py==0.4.1
astor==0.7.1
atomicwrites==1.2.1
attrs==18.2.0
colorama==0.3.9
cycler==0.10.0
gast==0.2.0
grpcio==1.15.0
kiwisolver==1.0.1
Markdown==2.6.11
matplotlib==2.2.3
more-itertools==4.3.0
numpy==1.14.5
Pillow==5.2.0
pluggy==0.7.1
protobuf==3.6.1
py==1.6.0
pyparsing==2.2.0
pytest==3.8.0
python-dateutil==2.7.3
pytz==2018.5
six==1.11.0
tensorboard==1.10.0
tensorboardX==1.4
tensorflow==1.10.0
termcolor==1.1.0
torch==0.4.1
torchvision==0.2.1
Werkzeug==0.14.1
```

In the docs (https://github.com/lanpa/tensorboardX/blob/master/docs/tutorial.rst#add-graph), it said:

> To draw the graph, you need a model m and an input variable t that have correct size for m. Let r = m(t), then please invoke writer.add_graph(m, r) to save the graph.

But when I see the source code, it is actually:

https://github.com/lanpa/tensorboardX/blob/9d2cbeb26778cc9784a6a7028e75b7fd0950ce87/tensorboardX/writer.py#L503

So the 2nd arg: ``input_to_model`` should be the input variable ``t``, not ``m(t)`` in the document. Because if I used ``m(t)`` in the code, I got error like:

```
C:\Python36\python.exe I:/github_repos/botrainer/train.py
Error occurs, No graph saved
Traceback (most recent call last):
  File "C:\Python36\lib\site-packages\tensorboardX\pytorch_graph.py", line 84, in graph
    trace, _ = torch.jit.get_trace_graph(model, args)
  File "C:\Python36\lib\site-packages\torch\jit\__init__.py", line 77, in get_trace_graph
    return LegacyTracedModule(f)(*args, **kwargs)
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 477, in __call__
    result = self.forward(*input, **kwargs)
  File "C:\Python36\lib\site-packages\torch\jit\__init__.py", line 109, in forward
    out = self.inner(*trace_inputs)
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 475, in __call__
    result = self._slow_forward(*input, **kwargs)
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 465, in _slow_forward
    result = self.forward(*input, **kwargs)
  File "I:/github_repos/botrainer/train.py", line 57, in forward
    output, self.hidden = self.lstm(input, self.hidden)
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 475, in __call__
    result = self._slow_forward(*input, **kwargs)
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 465, in _slow_forward
    result = self.forward(*input, **kwargs)
  File "C:\Python36\lib\site-packages\torch\nn\modules\rnn.py", line 178, in forward
    self.check_forward_args(input, hx, batch_sizes)
  File "C:\Python36\lib\site-packages\torch\nn\modules\rnn.py", line 130, in check_forward_args
    self.input_size, input.size(-1)))
RuntimeError: input.size(-1) must be equal to input_size. Expected 31, got 2

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "I:/github_repos/botrainer/train.py", line 238, in <module>
    writer.add_graph(rnn, r, verbose=False)
  File "C:\Python36\lib\site-packages\tensorboardX\writer.py", line 520, in add_graph
    self.file_writer.add_graph(graph(model, input_to_model, verbose))
  File "C:\Python36\lib\site-packages\tensorboardX\pytorch_graph.py", line 87, in graph
    _ = model(args)  # don't catch, just print the error message
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 475, in __call__
    result = self._slow_forward(*input, **kwargs)
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 465, in _slow_forward
    result = self.forward(*input, **kwargs)
  File "I:/github_repos/botrainer/train.py", line 57, in forward
    output, self.hidden = self.lstm(input, self.hidden)
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 475, in __call__
    result = self._slow_forward(*input, **kwargs)
  File "C:\Python36\lib\site-packages\torch\nn\modules\module.py", line 465, in _slow_forward
    result = self.forward(*input, **kwargs)
  File "C:\Python36\lib\site-packages\torch\nn\modules\rnn.py", line 178, in forward
    self.check_forward_args(input, hx, batch_sizes)
  File "C:\Python36\lib\site-packages\torch\nn\modules\rnn.py", line 130, in check_forward_args
    self.input_size, input.size(-1)))
RuntimeError: input.size(-1) must be equal to input_size. Expected 31, got 2

Process finished with exit code 1
```

However, if I use ``t`` as the 2nd arg, my code will work without this error. So can we fix the docs here: https://github.com/lanpa/tensorboardX/blob/master/docs/tutorial.rst#add-graph ?

**Comment by hsluoyz** [comment]
Thanks, @lanpa . I went to the deprecated site: https://tensorboard-pytorch.readthedocs.io/en/latest/tutorial.html because its ranking in Google is very high.

Can we remove that ``tensorboard-pytorch`` site entirely and only leave the ``tensorboardx`` site there? So most new users won't be misguided.

**Comment by lanpa** [comment]
Hi, I meant the `tutorial` is outdated. The link you provided should have same content as the link provided by me. Will find time to update the tutorial. It will be nice if you are willing to contribute to https://tensorboard-pytorch.readthedocs.io/en/latest/tutorial_zh.html

**Comment by lanpa** [comment]
fixed in #296. closing this.
